src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -21,21 +21,30 @@
 
 (def valid-commands-str (str/join ", " (sort (vals command-names))))
 
+(defn coerce-to-int
+  [s]
+  (if (number? s) s (Integer/parseInt s)))
```

Comment:
looks like my other comment got deleted, but I think `puppetlabs.kitchensink.core/parse-int` does basically the same thing as what we want here